Index: .idea/.gitignore
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
deleted file mode 100644
--- a/.idea/.gitignore	(revision 036a46b11b8f8c24829c16e9c792ea097ad0172d)
+++ /dev/null	(revision 036a46b11b8f8c24829c16e9c792ea097ad0172d)
@@ -1,10 +0,0 @@
-# Default ignored files
-/shelf/
-/workspace.xml
-# Ignored default folder with query files
-/queries/
-# Datasource local storage ignored files
-/dataSources/
-/dataSources.local.xml
-# Editor-based HTTP Client requests
-/httpRequests/
Index: ../.idea/.gitignore
===================================================================
diff --git a/../.idea/.gitignore b/../.idea/.gitignore
deleted file mode 100644
--- a/../.idea/.gitignore	(revision 036a46b11b8f8c24829c16e9c792ea097ad0172d)
+++ /dev/null	(revision 036a46b11b8f8c24829c16e9c792ea097ad0172d)
@@ -1,10 +0,0 @@
-# Default ignored files
-/shelf/
-/workspace.xml
-# Ignored default folder with query files
-/queries/
-# Datasource local storage ignored files
-/dataSources/
-/dataSources.local.xml
-# Editor-based HTTP Client requests
-/httpRequests/
Index: ../.idea/modules.xml
===================================================================
diff --git a/../.idea/modules.xml b/../.idea/modules.xml
deleted file mode 100644
--- a/../.idea/modules.xml	(revision 036a46b11b8f8c24829c16e9c792ea097ad0172d)
+++ /dev/null	(revision 036a46b11b8f8c24829c16e9c792ea097ad0172d)
@@ -1,8 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="ProjectModuleManager">
-    <modules>
-      <module fileurl="file://$PROJECT_DIR$/.idea/Blood-Donor Emergency System-fullstack.iml" filepath="$PROJECT_DIR$/.idea/Blood-Donor Emergency System-fullstack.iml" />
-    </modules>
-  </component>
-</project>
\ No newline at end of file
Index: ../.idea/vcs.xml
===================================================================
diff --git a/../.idea/vcs.xml b/../.idea/vcs.xml
deleted file mode 100644
--- a/../.idea/vcs.xml	(revision 036a46b11b8f8c24829c16e9c792ea097ad0172d)
+++ /dev/null	(revision 036a46b11b8f8c24829c16e9c792ea097ad0172d)
@@ -1,8 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="VcsDirectoryMappings">
-    <mapping directory="$PROJECT_DIR$" vcs="Git" />
-    <mapping directory="$PROJECT_DIR$/Backend" vcs="Git" />
-    <mapping directory="$PROJECT_DIR$/frontend" vcs="Git" />
-  </component>
-</project>
\ No newline at end of file
Index: src/test/java/com/serverproject/AppTest.java
===================================================================
diff --git a/src/test/java/com/serverproject/AppTest.java b/src/test/java/com/serverproject/AppTest.java
deleted file mode 100644
--- a/src/test/java/com/serverproject/AppTest.java	(revision 036a46b11b8f8c24829c16e9c792ea097ad0172d)
+++ /dev/null	(revision 036a46b11b8f8c24829c16e9c792ea097ad0172d)
@@ -1,38 +0,0 @@
-package com.serverproject;
-
-import junit.framework.Test;
-import junit.framework.TestCase;
-import junit.framework.TestSuite;
-
-/**
- * Unit test for simple App.
- */
-public class AppTest 
-    extends TestCase
-{
-    /**
-     * Create the test case
-     *
-     * @param testName name of the test case
-     */
-    public AppTest( String testName )
-    {
-        super( testName );
-    }
-
-    /**
-     * @return the suite of tests being tested
-     */
-    public static Test suite()
-    {
-        return new TestSuite( AppTest.class );
-    }
-
-    /**
-     * Rigourous Test :-)
-     */
-    public void testApp()
-    {
-        assertTrue( true );
-    }
-}
Index: ../.idea/Blood-Donor Emergency System-fullstack.iml
===================================================================
diff --git a/../.idea/Blood-Donor Emergency System-fullstack.iml b/../.idea/Blood-Donor Emergency System-fullstack.iml
deleted file mode 100644
--- a/../.idea/Blood-Donor Emergency System-fullstack.iml	(revision 036a46b11b8f8c24829c16e9c792ea097ad0172d)
+++ /dev/null	(revision 036a46b11b8f8c24829c16e9c792ea097ad0172d)
@@ -1,9 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<module type="JAVA_MODULE" version="4">
-  <component name="NewModuleRootManager" inherit-compiler-output="true">
-    <exclude-output />
-    <content url="file://$MODULE_DIR$" />
-    <orderEntry type="inheritedJdk" />
-    <orderEntry type="sourceFolder" forTests="false" />
-  </component>
-</module>
\ No newline at end of file
Index: ../.idea/misc.xml
===================================================================
diff --git a/../.idea/misc.xml b/../.idea/misc.xml
deleted file mode 100644
--- a/../.idea/misc.xml	(revision 036a46b11b8f8c24829c16e9c792ea097ad0172d)
+++ /dev/null	(revision 036a46b11b8f8c24829c16e9c792ea097ad0172d)
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="ProjectRootManager" version="2" project-jdk-name="openjdk-26" project-jdk-type="JavaSDK">
-    <output url="file://$PROJECT_DIR$/out" />
-  </component>
-</project>
\ No newline at end of file
